addons/mail/static/src/messaging/service/messaging_service/messaging_service.js

```diff
@@ -96,17 +98,11 @@ const MessagingService = AbstractService.extend({
      */
     destroy(...args) {
         this._super(...args);
-        this.env.messaging = undefined;
         this.env.bus.off('hide_home_menu', this);
         this.env.bus.off('show_home_menu', this);
         this.env.bus.off('will_hide_home_menu', this);
         this.env.bus.off('will_show_home_menu', this);
-        if (this.env.entities && this.env.entities.Entity) {
-            const { Entity } = this.env.entities;
-            while (Entity.all.length > 0) {
-                Entity.all[0].delete();
-            }
-        }
+        this.env.messaging.stop();
```

Comment:
`if (this.env.messaging)` necessary, destroy can be called before messaging is initialized